src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,178 @@
+package storage;
+
+import task.*;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.UnsupportedEncodingException;
+import java.util.Scanner;
+
+public class Storage {
+
+    private String directory;
+    private String fileName;
+
+    public Storage(String directory, String fileName) {
+        this.directory = directory;
+        this.fileName = fileName;
+    }
+    /**
+     * Saves the tasklist.
+     * Converts a tasklist into its string representation then saves it in a file
+     *
+     * @param taskList  the tasklist to be saved
+     */
+    public void save(TaskList taskList) {
```

Comment:
Perhaps the method name "save" could be renamed to "saveTaskList" or "saveToHarddisk".

Reply:
Changed to what you have suggested! Thank you!